rollup/conf/genesis.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This method enlightens me that maybe we can also reuse some other info (e.g., part of chunk-proposer and batch-proposer limits) from the genesis config.